students/grzegorz_bajorski/lesson_02_flow_control/chocolate_bar.py

```diff
@@ -0,0 +1,6 @@
+n = int(input())
+m = int(input())
+k = int(input())
+
+print ('YES')  if (k % m == 0 or k % n == 0) and k < m*n  else print('NO')
+
```

Comment:
Too many empty lines at the end of the file. According to PEP8 should be only one.